src/ReleaseHistory.md

```diff
@@ -5,6 +5,8 @@
 * BUGFIX: Fix `error ERR997.ExceptionLoadingPdb : '[filename]' was not evaluated because its PDB could not be loaded (E_PDB_NOT_FOUND).` when reading PE file built with `PDBPageSize:8192` or greater, by upgrading msdia140.dll from `14.27.28826.96` to `14.32.31326.0`. [685](https://github.com/microsoft/binskim/pull/685)
 * FEATURE: Add BA3031.EnableClangSafeStack, rename BA3030.UseCheckedFunctionsWithGcc to BA3030.UseGccCheckedFunctions [#663](https://github.com/microsoft/binskim/pull/663)
 * Bump Sarif.Sdk by updating submodule from [fc9a9df to 698adb6](https://github.com/microsoft/sarif-sdk/compare/fc9a9dfb865096b5aaa9fa3651854670940f7459...698adb6365a242c6bb75adde56e3bd4be39c21d7). [#674](https://github.com/microsoft/binskim/pull/674)
+* Introduce first performance rule BA6001.DisableIncrementalLinkingInReleaseBuilds [#667](https://github.com/microsoft/binskim/pull/667)
+* Introduce more performance rules BA6002, BA6004, BA6005, BA6006 [#691](https://github.com/microsoft/binskim/pull/691)
```

Comment:
>BA6002 [](http://example.com/codeflow?start=35&length=6)

Please add the dotted friendly names here just as we have with BA6001. Also, encapsulate all rule names in back-ticks for better markdown visualization.

`BA6001.DisableIncrementalLinkingInReleaseBuilds`

Reply:
Will do